### PR TITLE
Milestone2 completed.

### PR DESCRIPTION
**Pull request by belsman**
## In this iteration, we 

 - Created a directory structure for our bookstore app
 - Created Components: BooksList, BookForm, Book, and App
 - Displayed a list of books in our BooksList component
 - Composed the Components into an app
 - Created basic actions and reducers
 - Created a state with redux for our app

### CODE REVIEW

File: src/reducers/books.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actionTypes';
+
+const initialState = { books: [{ id: Math.floor(Math.random() * 100), title: 'The Dark Tower', category: 'Horror' }, { id: Math.floor(Math.random() * 10), title: 'Fantastic Beast', category: 'Sci-Fi' }] };
```
Comment (bellom) [inline src/reducers/books.js]: Well done guys,

According to the instruction on pathwright, you need to create the book object in `src/index.js`

image

Comment (bellom) [inline src/reducers/books.js]: ### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
{ id: uuid(), title: 'The Dark Tower', category: 'Horror' }
```

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </tbody>
+  </table>
+);
+
+const mapStateToProps = state => {
+  const { books } = state;
+  return books;
+};
+
+export default connect(mapStateToProps)(BooksList);
```
Comment (bellom) [inline src/containers/BooksList.js]: Let's arrange the code, this line should be the last line, for good code structure.

`export default connect(mapStateToProps)(BooksList);`

File: src/actions/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actionTypes';
+
+export const createBook = book => ({ // eslint-disable-line
```
Comment (bellom) [inline src/actions/index.js]: **Inline Comment:**
With the ESLint inline comment, we are indirectly changing the rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

Drop a comment if you're unable to fix it.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: :clap: 
Hello team :hand: 
Awesome job on this milestone and also a great job implementing the requested changes.
Good luck with the next one.

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**